src/services/user.service.js

```diff
@@ -31,6 +33,23 @@ class UserService {
     });
   }
 
+  /**
+   *
+   *
+   * @static
+   * @param {item} email
+   * @param {item} isVerified
+   * @returns {updated} @memberof UserService
+   */
+  static updateAccountStatus(email, isVerified) {
```

Comment:
Use the same function update user below(the functionalities are the same) you don't need to create a new one.